pure2.md

```diff
@@ -0,0 +1,535 @@
+        }
+    ],
+    "signed":
+    {
+        "_type": "Offline-Targets",
+        "expires": "2022-02-24T20:54:38Z",
+        "targets":
+        {
+            "firmware-acme-1.0.2.bin":
+            {
+                "hashes":
+                {
+                    "sha256": "994dc699d4fa98f8dd16d7a1abbd3cc7943c83e0f1aaab63a49e1f5c0aa8a363"
+                },
+                "length": 49827,
+                "custom":
+                {
+                    "name": "firmware-acme",
+                    "version": "1.0.2",
+                    "hardwareIds":
+                    [
+                        "acme-flibberator-NBB2"
+                    ],
+                    "targetFormat": "BINARY",
+                    "uri": null,
+                    "createdAt": "2021-04-05T14:24:00Z",
+                    "updatedAt": "2021-04-05T14:24:00Z",
+                }
+            },
+            "firmware-bravo-3.1.1.bin":
+            {
+                "hashes":
+                {
+                    "sha256": "5614fa9539dd64735ac5afe9f81a6fafb24998f32ef0eaa62d7e88e422272d2b"
+                },
+                "length": 1011928,
+                "custom":
+                {
+                    "name": "firmware-bravo",
+                    "version": "3.1.1",
+                    "hardwareIds":
+                    [
+                        "bravo-turboencabulator"
+                    ],
+                    "targetFormat": "BINARY",
+                    "uri": null,
+                    "createdAt": "2021-04-05T14:24:47Z",
+                    "updatedAt": "2021-04-05T14:24:47Z",
+                }
+            },
+        "version": 5
+    }
+}
+```
+
+#### metadata/director/offline-update-snapshot.json
+
+This sample offline-update snapshot metadata file indicates to the device that the latest version of offline-update-EMEA.json is 5, and the latest version of offline-update-APAC.json is 3 (and also indicates the hashes of those files, though this is not required by PURE-2). The device SHALL reject any other version of either of those files. It also SHALL reject any offline updates after 2021-11-21T15:38:10Z. Note that this doesn't mean that offline-update-EMEA.json version 5 and offline-update-APAC.json version 3 stop being valid on that date: if the device already has valid offline-update snapshot metadata with a higher version number than this (which might have been received from an OTA update or from another offline update), it can still use the versions of offline-update-EMEA.json and offline-update-APAC.json on this USB stick if the newer offline-update-snapshot.json indicates those are still the latest versions of those files.
```

Comment:
The USB stick reference comes a bit out of nowhere. I'd suggest using consistent terminology about known locations or portable storage media.

Reply:
Here (and above) you mention `offline-update-APAC.json`, but the actual example has `EMEA-premium.json`, which is also referred to above.

Reply:
I revamped this quite a bit, but I think it's a lot stronger now. I moved the text giving the example of how it could work to have a newer version of snapshot so that it's after the example file, and split it into several different sentences for clarity.